junit-platform-commons/src/main/java/org/junit/platform/commons/support/conversion/ConversionSupport.java

```diff
@@ -97,48 +47,49 @@ private ConversionSupport() {
 	 * type is a reference type
 	 *
 	 * @since 1.11
+	 * @see DefaultConversionService
```

Comment:
We should not link to a package-private type here as it will not be included in the published Javadoc.

```suggestion
```

Reply:
`DefaultConversionService` now takes over the detailed explanation in the Javadoc about the conversion strategy for string parameters. Would it make sense to make the class `public` and mark it with `@API(status = INTERNAL, since = "5.12")`?

Reply:
IIRC `DefaultArgumentConverter` needs to be `public` because it's accessed from other packages. I can see your point and I'd be okay with doing this for `DefaultConversionService`.